packages/fast-components-class-name-contracts-base/src/dialog/index.ts

```diff
@@ -2,7 +2,18 @@
  * The class name contract for the dialog component
  */
 export interface IDialogClassNameContract {
+    /**
+     * The root dialog component
+     */
     dialog: string;
+
+    /**
+     * The dialog backdrop
```

Comment:
You're mixing terminology here - maybe change to "The element that overlays the page while the modal is visible" or something to that effect